packages/blade/src/components/FileUpload/docs/FileUpload.stories.tsx

```diff
@@ -0,0 +1,727 @@
+import { useState } from 'react';
```

Comment:
in the example story I cant toggle some of the props.

Error text doesn't really show any error if I add it, upload type always remain single. Ideally if we are giving this playground then it should change on interaction otherwise disable that knob from being changed by the user

Reply:
@chaitanyadeorukhkar ErrorText will be displayed only when validationState is set to error.